src/tt_citites_skylines/core.rb

```diff
@@ -11,43 +11,32 @@ module TT::Plugins::CitiesSkylinesTools
 
 
   require "tt_citites_skylines/exporter.rb"
+  require "tt_citites_skylines/exporter_dae.rb"
   require "tt_citites_skylines/guide_grid.rb"
 
 
   unless file_loaded?(__FILE__)
+    menu = UI.menu("Plugins").add_submenu(PLUGIN_NAME)
+    toolbar = UI::Toolbar.new(PLUGIN_NAME)
 
     # Commands
-
-    cmd = UI::Command.new("Export Asset") {
-      self.export_asset
+    toolbaritems = ["Export CSL FBX", "Export CSL DAE", "Guide Grid Config", "Increase Subdivision Level", "Decrease Subdivision Level", "One Floor Up", "One Floor Down" ]
```

Comment:
I don't know about this - very hard to read this code. Goes way beyond 80 characters. (I should mention in the readme that the project follows GitHub Ruby Styleguide: https://github.com/styleguide/ruby)

Compact code isn't a goal in its self - readability is much more important.